Source/WebCore/animation/CSSPropertyAnimation.cpp

```diff
@@ -3737,7 +3737,6 @@ CSSPropertyAnimationWrapperMap::CSSPropertyAnimationWrapperMap()
         case CSSPropertyWebkitTapHighlightColor:
 #endif
         case CSSPropertyWebkitTextCombine:
-        case CSSPropertyWebkitTextDecoration:
```

Comment:
I expect you need to add `CSSPropertyTextDecoration` to `animatableShorthandProperties`.